cmd/kmagent/kmagent.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/kardianos/service"
+	bgsvc "github.com/kardianos/service"
+
+	"github.com/kloudmate/km-agent/internal/agent"
+)
+
+var logger bgsvc.Logger
+
+func main() {
+	var svcConfig = &bgsvc.Config{
+		Name:        "kmagent",
+		DisplayName: "KloudMate Agent",
+		Description: "KloudMate Agent for auto instrumentation",
+	}
+	prg, err := agent.NewKmAgentService()
+	if err != nil {
+		log.Fatal(err)
+	}
+	s, err := service.New(prg, svcConfig)
+	if err != nil {
+		log.Fatal(err)
+	}
+	logger, err = s.Logger(nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if len(os.Args) > 1 {
```

Comment:
@ansh-devs let's use https://github.com/urfave/cli
We'll need to support lot more complex command line arguments